.env.test

```
@@ -0,0 +1,6 @@
+VITE_SUPABASE_ANON_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im9oZmZmZ2RwdWFmbmFuemxuc3ZvIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDgwMDUxNzYsImV4cCI6MjA2MzU4MTE3Nn0.vqdP-kcc85nOGdnAeSxV2DNdAg5ejPZYne7r4A_9Ua8"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)